packages/eslint-config-airbnb-base/rules/imports.js

```diff
@@ -86,7 +86,7 @@ module.exports = {
 
     // disallow non-import statements appearing before import statements
     // https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/imports-first.md
-    // TODO: enable?
+    // TODO: enable
```

Comment:
actually since this is changing the guide, we should just go ahead and enable it now - i've got some breaking changes queued up anyway